osu.Game.Tournament/Screens/Editors/TournamentEditorScreen.cs

```diff
@@ -42,6 +68,7 @@ private void load()
                     RelativeSizeAxes = Axes.Both,
                     Anchor = Anchor.TopCentre,
                     Origin = Anchor.TopCentre,
+                    Padding = new MarginPadding { Bottom = backButton.Height },
```

Comment:
Don't think this is doing anything

Reply:
Without padding the rows will overlap with the back button, as demonstrated in [this video](https://streamable.com/0del3r). Adding the padding at the bottom makes sure the back button doesn't overlap with the rows. [Here](https://streamable.com/i56fmu) is another video to compare it against.

Reply:
I see. I've reapplied this to a more appropriate location (on the content flow itself).